ApplicationConfig/Providers/SessionInfo.php

```diff
@@ -29,7 +29,7 @@ class SessionInfo implements Provider
 
     public function __construct(
         SessionInterface $session,
-        CsrfTokenManagerInterface $csrfTokenManager,
+        CsrfTokenManagerInterface $csrfTokenManager = null,
```

Comment:
You cannot have a default value here. If you want to add `null` as default value, you must move the argument at the end.